package.json

```diff
@@ -85,7 +85,7 @@
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-promise": "6.0.1",
     "ethereum-waffle": "^3.3.0",
-    "hardhat": "^2.13.1",
+    "hardhat": "^2.17.2",
```

Comment:
just a quick check that everything else works with this version bump?

Reply:
ill return it